### ISSUE TITLE
decreased number of deaths in time series data for Germany

### ISSUE THREAD

**Issue by mponce0** [CSSEGISandData/COVID-19#2165]
I wrote an R package that uses the data reported from your repository, and one user noticed an irregularity with the time series data reported for Germany.

In particular, between the dates 2020-04-10 (with 2767) and 2020-04-11 (with 2736), ie. the number of reported deaths decreased.

Furthermore, a quick check shows that the following entries have irregularities with respect to the cumulative quantities reported in 

https://github.com/CSSEGISandData/COVID-19/blob/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_deaths_global.csv

ie.

         Province.State Country.Region
44 Prince Edward Island         Canada
45         Quebec         Canada
91                        Cyprus
107                       Finland
121                       Germany
131                       Iceland
132                         India
142                    Kazakhstan
183                   Philippines
195                        Serbia
198                      Slovakia

Is there something I may be missing?

Many thanks!
Marcelo

**Comment by therealcyberlord** [comment]
It might be a miscounting for Germany. I mean that can happen as well.

**Comment by mponce0** [comment]
It is not just Germany, all these entries show the same anomaly in the data at some point in time:

44 Prince Edward Island Canada
45 Quebec Canada
91 Cyprus
107 Finland
121 Germany
131 Iceland
132 India
142 Kazakhstan
183 Philippines
195 Serbia
198 Slovakia

**Comment by alfkoehn** [comment]
The error in the number of deaths for Germany for that date is already discussed in a few issues:

- https://github.com/CSSEGISandData/COVID-19/issues/2137
- https://github.com/CSSEGISandData/COVID-19/issues/2132

**Comment by tomo-one** [comment]
> 
> 
> It might be a miscounting for Germany. I mean that can happen as well.

Yes - but it should be explained and fixed ASAP. How can nations trust this numbers if nobody can explain or fix errors? A lot of intitutions an countries relay on this numbers.
